### PR TITLE
Various small editorial changes.

### PR DESCRIPTION
**Pull request by taylor-b**
Most notably:
- Mark the RTCP mux and bundle policy enum descriptions as
  non-normative. JSEP is the source of truth here, and goes into much
  more detail.
- Stop using all-caps BUNDLE.

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -336,25 +336,25 @@
           class="simple">
             <tbody>
               <tr>
-                <th colspan="2">Enumeration description</th>
+                <th colspan="2">Enumeration description (non-normative)</th>
```
Comment (alvestrand) [inline webrtc.html]: Suggest replacing this with "Enumeration description, summarized from the normative description in (JSEP-reference)".

**Comment by alvestrand** [comment]
Travis reacted to a duplicate definition. Please fix.

File: webrtc.html (hunk 1 above)
Comment (taylor-b) [inline webrtc.html]: I tried that, but the hyperlink ends up looking ugly on the blue background: [screenshot](http://i.imgur.com/5xbWaC9.png)

Also, the correct section of JSEP is already linked a couple lines above this, so it's a bit redundant anyway.

Any suggestions? My goal is to make it clear that the enum names are normative, but the meanings are not.

**Comment by taylor-b** [comment]
Fixed.

**Comment by aboba** [comment]
LGTM.

**Pull request merged**